docs/includes/swagger/cds_full.json

```diff
@@ -2225,7 +2225,7 @@
       },
       "x-conditional" : [ "additionalValue" ]
     },
-    "BankingProductLendingRate" : {
+    "BankingProductLendingRateV2" : {
```

Comment:
Incorrect version number detected. Should be V3 with backfill.
https://github.com/ConsumerDataStandardsAustralia/standards-maintenance/issues/311